force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -52,6 +94,16 @@ public class CollectionsTasks {
     List<String> codeTicks,
     List<Integer> priceTicks
   ) {
-    return 0;
+    
+    Integer stockPosition = 0;
+    Integer firstOccurence = codeTicks.indexOf(stock);
+
+    for (Integer i = firstOccurence; i < codeTicks.size(); i++) {
```

Comment:
Try to find a more efficient algorithm.
You could have an array of 10000000 elements and all of them with the stock tick - you'll have to iterate of the whole collection which is not efficient.

Another example:
N = 10000000 elements, first N -1 elements have the requires stock tick and the Nth element is a different stock tick.
Still, you'll have to iterate over almost whole collection of N.

Reply:
Updated. But I still think the previous solutions was more optimal. In your case it has sence, but imagine if there is an array with 100k elements, but only the first one is needed. So in this case the loop would iterate 100k times.